CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Minor: Update `CustomFieldInputSingleChoice` to select things
```

Comment:
```suggestion
  - Minor: Update `CustomFieldInputSingleChoice` to select choices
```

In the future, we'll probably use the changelog as a means of communicating changes to the rest of the teams. We should consider the language used when drafting the changelog items -- we want to ensure concise but descriptive context. In this case, we know we want the user to select a "choice".